export/index.html

```diff
@@ -377,6 +501,8 @@ <h2>Message log</h2>
         window.postMessage({
           "type": "INJECT_KEY_EXPORT_BUNDLE",
           "value": document.getElementById("key-export-bundle").value,
+          "keyFormat": document.getElementById("key-export-format").value,
+          "publicKey": document.getElementById("key-export-public-key").value,
```

Comment:
We could potentially derive the public key from the private key in the iframe to avoid this extra param, but this means supporting ed25519. Not sure it's worth the complexity (webcrypto doesn't support it natively)

Reply:
how do they end up getting the public key? is it sent down in the bundle?

Reply:
yep, currently it'd get sent down from the js sdk so they would need to fetch the private key's public key when the user exports it... i can look into supporting the curve because it would be nice to only ask for the key format field. having two optional fields feels like extra education for devs.

Reply:
@r-n-o @timurnkey i haven't had time to look into this today, but wdyt about adding https://www.npmjs.com/package/@noble/ed25519 as an ES module just to derive the public key for a private key when `solana` is passed as the key format? not ideal to be using a CDN but it could be a short term solution until we can implement this ourselves (and/or when webcrypto supports it) that allows for a long term solution of not needing to pass the public key param

Reply:
Yeah we could pull a dependency for it for sure. We could also grab just the code we need and inline it with attribution. At the end of the day what we need is "just" a bigInt multiplication / exponentiation to go from private key (int) to public key (point on the curve).

Reply:
+1 to inlining since it's a sensitive flow.

Would this dependency be an attacker vector? idk 🤷 Are there ways a ES module dependency could eject data out of the iframe (ajax, events, extensions)?

Reply:
I agree that inlining + attribution is the ideal solution. Since we need to get this fix out tomorrow (this would be our third delay otherwise), wdyt about shipping a temporary fix (https://github.com/tkhq/frames/pull/28) that uses a ES module dependency and then Monday I can start working on the inline replacement.